<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test_JSON</class>
 <widget class="QWidget" name="Test_JSON">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>265</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="tj_splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="ShowJSON" name="ShowJSON_widget" native="true"/>
     <widget class="QTextEdit" name="te_result">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Liberation Sans'; font-size:10pt;&quot;&gt;{&amp;quot;status&amp;quot;:{&amp;quot;timestamp&amp;quot;:&amp;quot;2020-07-01T22:57:39.423Z&amp;quot;,&amp;quot;error_code&amp;quot;:0,&amp;quot;error_message&amp;quot;:null,&amp;quot;elapsed&amp;quot;:26,&amp;quot;credit_count&amp;quot;:1,&amp;quot;notice&amp;quot;:null},&amp;quot;data&amp;quot;:{&amp;quot;BTC&amp;quot;:{&amp;quot;id&amp;quot;:1,&amp;quot;name&amp;quot;:&amp;quot;Bitcoin&amp;quot;,&amp;quot;symbol&amp;quot;:&amp;quot;BTC&amp;quot;,&amp;quot;slug&amp;quot;:&amp;quot;bitcoin&amp;quot;,&amp;quot;num_market_pairs&amp;quot;:8306,&amp;quot;date_added&amp;quot;:&amp;quot;2013-04-28T00:00:00.000Z&amp;quot;,&amp;quot;tags&amp;quot;:[&amp;quot;mineable&amp;quot;,&amp;quot;state-channels&amp;quot;,&amp;quot;sha-256&amp;quot;,&amp;quot;privacy&amp;quot;,&amp;quot;pow&amp;quot;,&amp;quot;store-of-value&amp;quot;],&amp;quot;max_supply&amp;quot;:21000000,&amp;quot;circulating_supply&amp;quot;:18420268,&amp;quot;total_supply&amp;quot;:18420268,&amp;quot;is_active&amp;quot;:1,&amp;quot;platform&amp;quot;:null,&amp;quot;cmc_rank&amp;quot;:1,&amp;quot;is_fiat&amp;quot;:0,&amp;quot;last_updated&amp;quot;:&amp;quot;2020-07-01T22:56:33.000Z&amp;quot;,&amp;quot;quote&amp;quot;:{&amp;quot;USD&amp;quot;:{&amp;quot;price&amp;quot;:9232.22783688,&amp;quot;volume_24h&amp;quot;:15957216692.0124,&amp;quot;percent_change_1h&amp;quot;:-0.0745469,&amp;quot;percent_change_24h&amp;quot;:0.870893,&amp;quot;percent_change_7d&amp;quot;:-0.65612,&amp;quot;market_cap&amp;quot;:170060110992.3899,&amp;quot;last_updated&amp;quot;:&amp;quot;2020-07-01T22:56:33.000Z&amp;quot;}}}}}&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_test">
       <property name="text">
        <string>test</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShowJSON</class>
   <extends>QWidget</extends>
   <header>showjson.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
